components/nds-a-z-list/src/nds-a-z-list.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module "@nice-digital/nds-a-z-list" {
+	import React = require("react");
+
+	export interface AZListProps {
+		alphabet: React.ElementType;
+		children: React.ReactNode;
+		className?: string;
+		[prop: string]: unknown;
+	}
+
+	export interface AZListItemProps {
+		title: string;
```

Comment:
These differ from the proptypes e.g. missing `id` and `className`

Reply:
Thanks - fixing this